scripts/create_xcframeworks.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# Shebang to explicitly use bash and not break on macOS boxes with zsh as default shell.
+#
+# After that you usually describe what this script is for and how to use it. In particular,
+# what it expects the current directory to be. (That's more important if it's it "scripts".)
+
+set -eu # common flags to ensure that the shell does not ignore failures
+
+BUILD_PATH=build
+output_dir=$BUILD_PATH/xcf_output
+
+cd ..
+repo_root_path=$(pwd)
```

Comment:
Hm... I'd argue it would be easier to just expect the script to be invoked as `scripts/create_xcframeworks.sh` from the project root. That's more natural for CI pipelines, etc. That's what relative path in `BUILD_PATH` expects too.